src/pd/async/util.rs

```diff
@@ -102,6 +164,12 @@ impl LeaderClient {
 
         {
             let mut inner = self.inner.wl();
+            let (tx, rx) = client.region_heartbeat();
+            inner.hb_sender = Either::Left(Some(tx));
+            if let Either::Right(ref mut task) = inner.hb_receiver {
+                task.unpark();
```

Comment:
If hb_receiver is a Task, L83 has been called, call `unpark` to notify the receiver poller.